### DISCUSSION TITLE
Background image : blur and ideal dimension

### DISCUSSION THREAD

**Discussion by niltied** [hugo-themes/toha#586] [Q&A] [answered]
### Question

Hi Emruz,

I would like to know if it's possible to disable easily the blur effect applied on the background image ? Which exactly instructions CSS to comment and which file ? (home.css ?)

Also, I've a nice image (6000px by 4000px). It's too big naturaly, but if I resize it, which ideal size to do ?

Thank U so much

Nicolas

**Comment by hossainemruz** [comment, marked as answer]
You have to change here: https://github.com/hugo-toha/toha/blob/a22474538d78e8f73339977de86f2b6408cf0c8e/static/css/sections/home.css#L17

**Comment by niltied** [comment]
OK thank.
